tool/jt.rb

```diff
@@ -2187,6 +2187,7 @@ def check_native_configuration
       https://github.com/oracle/truffleruby/blob/master/doc/user/installing-libssl.md
       https://github.com/oracle/truffleruby/blob/master/doc/user/installing-llvm.md
       https://github.com/oracle/truffleruby/blob/master/doc/user/installing-zlib.md
+      https://github.com/oracle/truffleruby/blob/master/doc/user/polyglot.md
```

Comment:
Not now, but we probably could allow all `/^https:\/\/github.com\/oracle\/truffleruby/` links